sources/co/bogota_addressforall.json

```
@@ -0,0 +1,33 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "CO-DC",
+            "country": "Colombia",
+            "region": "Distrito Capital de Bogotá"
+        },
+        "country": "co",
+        "state": "dc"
+    },
+    "schema": 2,
+    "layers": {
+        "addresses": [
+            {
+                "name": "state",
+                "data": "https://addressforall-my.sharepoint.com/personal/operacao_addressforall_org/_layouts/15/download.aspx?share=EfECM6Ea5LJDlSWmlVA4l0oBHa_ufTUdHQCoFWws4qZsUA",
+                "website": "https://github.com/digital-guard/preserv-CO/tree/main/data/DC/Bogota/_pk0001.01",
+                "protocol": "http",
+                "license": {
+                    "url": "https://creativecommons.org/licenses/by/4.0/",
+                    "attribution": true,
+                    "attribution name": "CC-BY-4.0",
+                    "share-alike": true
```

The `CC-BY-4.0` license does not require share-alike. Can you set this to false?

```suggestion
                    "share-alike": false
```